frontend/src/components/AudioPlayer/index.js

```diff
@@ -150,7 +60,9 @@ function Audio({ songs, setAudioElement }, props) {
         src={songAudio?.url}
         header={songAudio?.title || "No title selected"}
         volume=".25"
-        onClick={handlePlayPause}
+        onPlay={() => dispatch(syncPlayerStatus('play'))}
+        onPause={() => dispatch(syncPlayerStatus('pause'))}
+        onError={() => dispatch(syncPlayerStatus('error'))}
```

Comment:
Whenever the audio player element emits one of these media events, dispatch an action to update the status in the audio store.